src/main/scala/tech/cryptonomic/nautilus/metering/repositories/InfluxDbRepo.scala

```diff
@@ -126,11 +134,13 @@ class InfluxDbRepoImpl(cfg: InfluxDbConfig)(
       }
 
   /** Reads amount of hits per IP grouped by 24hour intervals */
-  override def get24hIp(apiKeys: List[String]): Future[List[IpCQ]] =
+  override def get24hIp(apiKeys: List[String], from: Option[Long]): Future[List[IpCQ]] =
```

Comment:
secondary remark: why aren't these methods named consistently with the  corresponding columns?

daily, fiveMinutes, ...

Reply:
ok, renamed